### PR TITLE
Add SQS test

### PR DESCRIPTION
**Pull request by darkdh**
fix https://github.com/brave/sync/issues/220

### CODE REVIEW

**Comment by diracdeltas** [comment]
travis failing :O

File: test/client/requestUtil.js [hunk 1]
```diff
@@ -30,10 +30,13 @@ test('client RequestUtil', (t) => {
...
         const requestUtil = new RequestUtil(args)
-        t.pass('can instantiate requestUtil')
-        t.test('prototype', (t) => {
-          testPrototype(t, requestUtil, data.keys)
-        })
+        requestUtil.createAndSubscribeSQS('0')
+          .then(()=> {
+            t.pass('can instantiate requestUtil')
+            t.test('prototype', (t) => {
+            testPrototype(t, requestUtil, data.keys)
```
Comment (diracdeltas) [inline test/client/requestUtil.js]: needs indentation

**Comment by darkdh** [comment]
seems like failing with npm audit. tests and lints are both fine

File: test/client/requestUtil.js [hunk 2]
```diff
@@ -175,12 +178,102 @@ test('client RequestUtil', (t) => {
               t.deepEquals(s3Record.deviceId, record.deviceId, `${t.name}: deviceId`)
               t.deepEquals(s3Record.objectId, record.objectId, `${t.name}: objectId`)
               t.deepEquals(s3Record.historySite, record.historySite, `${t.name}: historySite.location`)
-              testCanDeleteHistorySites(t)
```
Comment (diracdeltas) [inline test/client/requestUtil.js]: why was this moved into a callback? seems that the two tests should be independent

Comment (darkdh) [reply test/client/requestUtil.js]: it is called after `testCanDeleteBookmarks`

**Comment by diracdeltas** [comment]
https://github.com/brave/sync/issues/232 is the vuln issue

**Comment by SergeyZhukovsky** [comment]
++ on the changes. We have to fix the travis error as autodeploy will be broken from that point.
